package.json

```diff
@@ -12,13 +12,14 @@
     "devnet": "ENABLE_MAINNET_FORKING=true NO_RESET=true npx hardhat node",
     "console:rinkeby": "npx hardhat console --network rinkeby",
     "console:ropsten": "npx hardhat console --network ropsten",
-    "clean": "rm -rf build && rm -rf out",
-    "test": "forge test",
+    "clean": "forge clean",
```

Comment:
does that also clean the hardhat builds files?

Reply:
No good point, have added a `rm -rf artifacts`